File: mbw/src/main/java/com/mycelium/wallet/activity/modern/adapter/holder/AccountViewHolder.java
package com.mycelium.wallet.activity.modern.adapter.holder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.ProgressBar;
import android.widget.TextView;

import com.mycelium.wallet.R;


public class AccountViewHolder extends RecyclerView.ViewHolder {
    public View llAddress;
    public TextView tvLabel;
    public TextView tvWhatIsIt;
    public ImageView icon;
    public TextView tvAddress;
    public TextView tvAccountType;
    public TextView tvBalance;
    public TextView backupMissing;
    public TextView tvProgress;
    public View tvProgressLayout;
    public ImageView ivWhatIsSync;
    public ProgressBar progressBar;

    public View tvTraderKey;

    public AccountViewHolder(View itemView) {
        super(itemView);
        llAddress = itemView.findViewById(R.id.llAddress);
        tvLabel = itemView.findViewById(R.id.tvLabel);
        tvWhatIsIt = itemView.findViewById(R.id.tvWhatIsIt);
        icon = itemView.findViewById(R.id.ivIcon);
        tvAddress = itemView.findViewById(R.id.tvAddress);
        tvAccountType = itemView.findViewById(R.id.tvAccountType);
        tvBalance = itemView.findViewById(R.id.tvBalance);
        backupMissing = itemView.findViewById(R.id.tvBackupMissingWarning);
        tvTraderKey = itemView.findViewById(R.id.tvTraderKey);
        tvProgress = itemView.findViewById(R.id.tvProgress);
        tvProgressLayout = itemView.findViewById(R.id.tvProgressLayout);
        ivWhatIsSync = itemView.findViewById(R.id.ivWhatIsSync);
        progressBar = itemView.findViewById(R.id.progress_bar);
    }
}

File: mbw/src/main/java/com/mycelium/wallet/activity/modern/adapter/holder/AccountViewHolder.kt
package com.mycelium.wallet.activity.modern.adapter.holder

import android.view.View
import android.widget.ImageView
import android.widget.ProgressBar
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.mycelium.wallet.databinding.RecordRowBinding

class AccountViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    val binding = RecordRowBinding.bind(itemView)
    val llAddress: View = binding.llAddress
    val tvLabel: TextView = binding.tvLabel
    val tvWhatIsIt: TextView = binding.tvWhatIsIt
    val icon: ImageView = binding.ivIcon
    val tvAddress: TextView = binding.tvAddress
    val tvAccountType: TextView = binding.tvAccountType
    val tvBalance: TextView = binding.tvBalance
    val backupMissing: TextView = binding.tvBackupMissingWarning
    val tvProgress: TextView = binding.tvProgress
    val layoutProgressTxRetreived: View = binding.progressTxRetreived
    val tvProgressLayout: View = binding.tvProgressLayout
    val ivWhatIsSync: ImageView = binding.ivWhatIsSync
    val progressBar: ProgressBar = binding.progressBar
    val lastSyncState: View = binding.lastSyncState
    val tvTraderKey: View = binding.tvTraderKey
}